doris/uninstall.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+./${DORIS_PACKAGE}/be/bin/stop_be.sh
```

Comment:
Since this PR adds a script `start.sh`, please
- either move l. 2-3 into a new script `stop.sh`, or
- remove `start.sh` and move its content into `install.sh`.

Reply:
add stop.sh now